sabelotodo/tests/test_routes.py

```diff
@@ -0,0 +1,87 @@
+from sabelotodo.models import Item
+from dataclasses import asdict
+
+import json
+
+
+def test_create_item(_db):
+    """ Verify that database works"""
+    name = 'TestItem'
+    done = True
+    i = Item(name=name, order=0, done=done)
+
+    _db.session.add(i)
+    _db.session.commit()
+
+    item = Item.query.filter_by(name=name).first()
+    assert i == item
```

Comment:
I'm not confident that `==` in python performs deep equality; ie, that it checks that every property of the object is the same, which is why I picked `done` as a thing to test on (and probably should have tested other properties as well. If you're sure that `==` performs deep equality in python, let's keep this, if not, let's go back to how it was? (I'm used to JavaScript which most definitely does *not* do deep equality in this case.

Reply:
Ooh, good call. Python objects can define how comparison operators treat them. But `Item` is a `@dataclass` class, and it looks like those compare equal if all their fields are equal -- which I think is what we want.